<launch>

	<!-- start  marker_server_node -->
	<node pkg="swarm_contol" name="interactive_goals_node" type="marker_server" 	respawn="true" output="screen">
		<!--remap from="/goal_pose" to="/geo/goal_pose"/-->
	</node>

	<node pkg="swarm_contol" name="swarm_marker_node" type="swarm_marker.py" 	respawn="true" output="screen">
		<param name="param" value="$(find swarm_contol)/param/config.yaml"/>
		<param name="max_vel" value="0.3"/>
	</node>

	<!--rviz-->
	<node name="rviz" pkg="rviz" type="rviz" required="true"
		args="-d $(find swarm_contol)/rviz/rviz.rviz" />
</launch>
